src/components/InputProblem.tsx

```diff
@@ -24,16 +24,17 @@ export const InputProblem = ({
   const [retriesAllowed, setRetriesAllowed] = useState(0)
   const [inputDisabled, setInputDisabled] = useState(false)
   const [feedback, setFeedback] = useState('')
+  const NUMERIC_INPUT_ERROR = 'Enter numeric values only'
 
   const schema = (): Yup.SchemaOf<InputSchema> => {
     if (comparator.toLowerCase() === 'integer') {
       return Yup.object({
-        response: Yup.number().integer('Please provide an Integer').typeError('Please provide an Integer').required('Please provide an Integer')
+        response: Yup.number().integer(NUMERIC_INPUT_ERROR).typeError(NUMERIC_INPUT_ERROR).required(NUMERIC_INPUT_ERROR)
       })
     }
     if (comparator.toLowerCase() === 'float') {
       return Yup.object({
-        response: Yup.number().typeError('Please provide an number').required('Please provide a number')
+        response: Yup.number().typeError(NUMERIC_INPUT_ERROR).required(NUMERIC_INPUT_ERROR)
       })
     }
     return Yup.object({
```

Comment:
I'm not sure I completely understand the question -- I believe the PR as you have it is correct in terms of implemented the feedback requested by UX for both `integer` and `float` validation. I think the concern with the `integer` comparator was more around making sure that if a user puts `abc` it will say `Enter numeric values only`. If they were to enter `1.000` then the validation should pass, but it won't if they enter `1.1` (you can confirm, but I'm fairly certain the `yup` validation gripes if there's a fractional component in the numeric value, not just if a decimal exists since `1 === 1.0`)